### PR TITLE
Update manager.py

### PR DESCRIPTION
**Pull request by gvancuts**

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Review by NanlinXie — APPROVED** [review]

File: misc/config_tools/configurator/packages/configurator/thirdLib/manager.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def main():
     library_json = Path(__file__).parent / 'library.json'
 
     parser = argparse.ArgumentParser(
-        description='ACRN Configurator third part library manager.'
+        description='ACRN Configurator third party library manager.'
```
Comment (dbkinder) [inline misc/config_tools/configurator/packages/configurator/thirdLib/manager.py:113]: ThirdLib/library.json has the same misspelling  (ACRN Third Part Library List)

Comment (gvancuts) [reply misc/config_tools/configurator/packages/configurator/thirdLib/manager.py:113]: > ThirdLib/library.json has the same misspelling (ACRN Third Part Library List)

That's true for many files from this new configurator.

**Comment by gvancuts** [comment]
> @NanlinXie Where does this thirdLib code come from? It has no copyright or licensing header.

That's true for many files from this new configurator.

**Comment by szhen11** [comment]
@gvancuts Please add Tracked-On info into commit message to let TRACKED_ON_TEST check passed.

**Pull request closed without merging**